### PR TITLE
Copy updates on /security

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Copy updates on /security
- Note: *there is an outstanding question about a link to the wiki*

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security
- Compare to the [copy doc](https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit#)

## Issue / Card

Fixes [#4279](https://github.com/canonical-web-and-design/web-squad/issues/4279)

### CODE REVIEW

**Comment by petesfrench** [comment]
@pmahnke I can't see any comments of what was changed in the doc

**Comment by petesfrench** [comment]
Demo: https://ubuntu-com-10084.demos.haus/

**Comment by pmahnke** [comment]
@petesfrench so you will either have to compare all the text, or use the version history to go back before I resolved them.  Sorry, I thought you guys were resolving as you went these days.

File: templates/security/index.html [hunk 1]
```diff
@@ -435,28 +430,40 @@ <h2>Helping you manage security</h2>
...
+      <p>The Canonical Livepatch service eliminates the need for unplanned maintenance windows for high and critical severity kernel vulnerabilities by patching the Linux kernel while the system runs. Reduce fire drills while keeping uninterrupted service with Canonical Livepatch service for up to ten years.</p>
+      <p>
+        <a href="/security/livepatch">
+          Learn more about Canonical's Livepatch Service&nbsp;&rsaquo;
+        </a>
+      </p>
     </div>
     <div class="col-6 p-card">
       <h3>Be compliant and FIPS certified</h3>
       <p>Canonical has received FIPS 140-2, Level 1 certification for cryptographic modules in Ubuntu 18.04 LTS and Ubuntu 16.04 LTS and Common Criteria EAL2 certification for Ubuntu 16.04 LTS. These certifications, alongside further security measures and achievements, enable organisations to meet compliance requirements within regulated industries.</p>
```
Comment (petesfrench) [inline templates/security/index.html:443]:
```suggestion
      <p>Developing and running workloads for high security and government regulated environments requires a long and expensive validation process. Reduce your accreditation timeline and pass on your validation costs with the FIPS 140 and Common Criteria certifications available with Ubuntu Advantage and Pro.</p>
```
Does not match copydoc

File: templates/security/index.html [hunk 2]
```diff
@@ -336,8 +332,7 @@ <h2 class="p-muted-heading u-align--center">Ubuntu is trusted by</h2>
     <div class="col-7">
       <h2>Find out why the UK Government puts Ubuntu in first place for security</h2>
       <p><abbr title="Communications-Electronics Security Group">CESG</abbr>, the security arm of the UK government rated Ubuntu as the most secure operating system of the 11 they tested.</p>
-      <p>For the first time, both a <abbr title="Defence Industry Security Association">DISA</abbr> approved <abbr title="Security Technical Implementation Guide">STIG</abbr> and a <abbr title="Center for Internet Security">CIS</abbr> Benchmark are available for Ubuntu 16.04 LTS. This is in addition to the CIS Benchmark already available for 14.04 LTS.</p>
-      <a class="p-button--neutral" href="/blog/ubuntu-scores-highest-in-uk-gov-security-assessment"><span class="p-link--external">Read the <span class="u-off-screen">UK Gov Report Summary </span>case study</span></a>
+      <p><a class="p-button--neutral" href="/blog/ubuntu-scores-highest-in-uk-gov-security-assessment"><span class="p-link--external">Read the <span class="u-off-screen">UK Gov Report Summary </span>case study</span></a></p>
     </div>
     <div class="col-5 u-align--center u-vertically-center">
       {{
```
Comment (petesfrench) [inline templates/security/index.html:338]: I think just a u-hide--small is needed on this image

File: templates/security/index.html [hunk 3]
```diff
@@ -477,7 +484,7 @@ <h2>Ubuntu Advantage</h2>
         <li class="p-list__item is-ticked">Knowledge Base: a private archive of expert-written articles and tutorials</li>
         <li class="p-list__item is-ticked">Support: phone and web-based support at multiple service levels</li>
       </ul>
-      <p><a href="/advantage" class="p-button--neutral">Purchase Ubuntu Advantage</a></p>
+      <p><a href="/advantage" class="p-button--positive">Purchase Ubuntu Advantage</a></p>
```
Comment (petesfrench) [inline templates/security/index.html:487]:
```suggestion
      <p><a href="/advantage" class="p-button--positive">Visit the Ubuntu Advantage store</a></p>
```
Doesn't match copydoc

**Review by petesfrench — CHANGES_REQUESTED** [review]
Just a couple differences from the copydoc.

Including two things I could not comment on in the code:
line 130 - link does not match the copydoc
line 493 - doesn't exist in copydoc

File: templates/security/index.html (hunk 3 above)
Comment (pmahnke) [reply templates/security/index.html:487]: I updated the copy doc in this case... someone forgot.

File: templates/security/index.html (hunk 2 above)
Comment (pmahnke) [reply templates/security/index.html:338]: cool

File: templates/security/index.html (hunk 1 above)
Comment (pmahnke) [reply templates/security/index.html:443]: good catch, that change wasn't highlighted

**Comment by pmahnke** [comment]
@petesfrench ready for a re-review

**Comment by petesfrench** [comment]
LGTM :+1: I made one small  grammar change in the doc

**Comment by pmahnke** [comment]
@petesfrench can you approve it as well?

**Review by petesfrench — APPROVED** [review]

**Pull request merged**